lib/tasks/doctor/index.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
```

Comment:
why does this file need to exist, it seems like just `glob('tasks/doctor')` is sufficient for generating this list.